src/server_manager/messages/master_messages.json

```diff
@@ -452,15 +452,15 @@
     "description": "This instruction is a clarification regarding the title of the billing page."
   },
   "gcp_billing_action": {
-    "message": "OPEN",
+    "message": "Open",
     "description": "Button label for a button that opens a the billing page in the web browser"
   },
   "gcp_billing_body": {
     "message": "Waiting for you to add a billing account on Google Cloud",
     "description": "Body text on the billing page"
   },
   "gcp_billing_refresh": {
-    "message": "CHECK AGAIN",
+    "message": "Check Again",
```

Comment:
Let's put this functionality in the Next button

Reply:
Done